### PR TITLE
Add explanation to php for `docker-php-source`

### PR DESCRIPTION
**Pull request by yosifkit**
Depends on https://github.com/docker-library/official-images/pull/1942

### CODE REVIEW

File: php/content.md [hunk 1]
```diff
@@ -68,6 +68,15 @@ Where `src/` is the directory containing all your php code and `config/` contain
 
 We provide the helper scripts `docker-php-ext-configure`, `docker-php-ext-install`, and `docker-php-ext-enable` to more easily install PHP extensions.
 
+In order to keep the images smaller, the php source is kept in a compressed tar file. To facilitate linking of the php source with any extension, we also provide the helper script `docker-php-source` to easily extract the tar or delete the extracted source. Note: if you do use `docker-php-source` to extract the source, be sure to delete it in the same layer of the docker image.
```
Comment (tianon) [inline php/content.md]: s/`the php source`/`PHP's source`/g

File: php/content.md [hunk 2]
```diff
@@ -68,6 +68,15 @@ Where `src/` is the directory containing all your php code and `config/` contain
...
+
+```Dockerfile
+FROM php:5-apache
```
Comment (tianon) [inline php/content.md]: This should be `5.6` like all the rest so that we encourage users to use `X.Y`. :+1:

Comment (tianon) [inline php/content.md]: :astonished: the `fpm` example below is using `:5` too :disappointed:

Comment (yosifkit) [inline php/content.md]: :open_mouth: you found where I took it from.

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
Your changes look good to me. :+1:

**Pull request merged**